packages/custom_lint_core/lib/src/configs.dart

```diff
@@ -35,13 +37,18 @@ class CustomLintConfigs {
     final include = yaml['include'] as Object?;
     var includedOptions = CustomLintConfigs.empty;
     if (include is String) {
-      final includeAbsolutePath = absolute(
-        analysisOptionsFile.parent.path,
-        include,
-      );
-      includedOptions = CustomLintConfigs.parse(
-        analysisOptionsFile.provider.getFile(includeAbsolutePath),
-      );
+      final includeAbsolutePath = include.startsWith('package:')
+          ? Isolate.resolvePackageUriSync(Uri.parse(include))?.toFilePath()
+          : absolute(
+              analysisOptionsFile.parent.path,
+              include,
+            );
```

Comment:
I believe you do not need to check if it is a package import since `Isolate.resolvePackageUriSync` will return the unmodified Uri if it is not. I believe `absolute` has similar logic - if it is already absolute it won't modify it.